### DISCUSSION TITLE
Consider dropping `libgfold` and collapsing into one crate

### DISCUSSION THREAD

**Discussion by nickgerace** [nickgerace/gfold#268] [Ideas]
As of the time of writing, [libgfold](https://crates.io/crates/libgfold)'s only user is gfold according to [crates.io](https://crates.io/crates/libgfold). 

The purpose of libgfold was to expose gfold internals for external consumption. Given the steady popularity of gfold itself, the lack of external users of libgfold and the potential de-bloating opportunities that using a single crate offers (e.g. using `anyhow` everywhere), it might be worth consolidating everything down to one crate.

**Comment by nickgerace** [comment]
@uncenter do you have any thoughts here? I know you had other things you wanted to contribute (thank you and no pressure to do so!), so I wanted to check with you.

**Comment by uncenter** [comment]
This sounds like a good idea yep! I can try consolidating it unless you have some specific details in mind.

**Comment by nickgerace** [comment]
Closed by https://github.com/nickgerace/gfold/pull/271
